tests/integration/federation/test.sh

```diff
@@ -50,10 +50,13 @@ function check_init() {
   fi
 }
 
-function configure() {
+function configure_trust_zones() {
   ./cofidectl trust-zone add $TRUST_ZONE_1 --trust-domain $TRUST_DOMAIN_1 --kubernetes-context $K8S_CLUSTER_1_CONTEXT --kubernetes-cluster $K8S_CLUSTER_1_NAME --profile kubernetes
   ./cofidectl trust-zone add $TRUST_ZONE_2 --trust-domain $TRUST_DOMAIN_2 --kubernetes-context $K8S_CLUSTER_2_CONTEXT --kubernetes-cluster $K8S_CLUSTER_2_NAME --profile kubernetes
-  ./cofidectl federation add --from $TRUST_ZONE_1 --to $TRUST_ZONE_2
```

Comment:
I was keeping the old from/to arguments around to check that they still work until we remove them

Reply:
The CI is failing (but not locally) so I'm trying to see if this explains it

Reply:
The federation test can be a bit flakey